### ISSUE TITLE
Arm Inverse Kinematics Mode

### ISSUE THREAD

**Issue by superdan-t** [BinghamtonRover/RoverSystem#242]
## Description
The arm must be controllable using either joint mode or inverse kinematics mode. Arm control should be active in the Arm focus mode and there should be a controller button to toggle between joints and IK. Arm movement should also support magnitude (ex. full joystick movement vs. partial).

In the current setup, joints may be selected and rotated clockwise or counterclockwise. This will become joint mode. In the new IK mode, movements are in directions (up, down, left, right, forward, back) and do not apply to specific joints. These will be sent over the CAN bus in the future. For now, movements may be log entries.
## Scope and Requirements
- Create new or modify existing network packets to handle both joint movements and IK movements. A magnitude variable will need to be added.
- Allow users to toggle between Joint and IK mode in the Arm Focus Mode. Toggling should be done on the base station side only. The subsystems computer should always be able to handle both packets. This should be a controller key if there is a free key. If not, use a keyboard key. The joysticks have buttons, too, which may be an intuitive way to switch.
- Add a control scheme for arm movements. Issue #226 and #233 describe a potential setup. The controls may need revising once we have feedback from operating experience.
- Sending arm commands is out of scope since the arm subsystem and CAN bus is under development. The subsystem computer event handler may be a skeleton with `TODO` comments for now. ROCS will likely be replaced entirely.

## Future Requirements
These items are not part of the core task and aren't required in this iteration. If you have extra time to implement them, feel free. Otherwise, we can come back once the software team's essential tasks are done.
- Graphical control mode indicator (`Arm Control: IK or Joint` or similar). Display selected joint name when in Joint mode.
- Update the arm focus mode help menu with the new controls. The PR for #233 is still open. It should be merged before getting to this task, but if not, this will be done on 233 instead.)
